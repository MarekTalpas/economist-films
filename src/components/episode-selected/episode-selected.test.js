// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import EpisodeSelected from './episode-selected';

jest.mock('./episode-description/episode-description', () =>
  jest.fn(() => <div>EpisodeDescription</div>),
);

test('EpisodeSelected renders with data', () => {
  const tree : string = renderer.create(<EpisodeSelected
    id={3}
    url="url xyz"
    title="title xyz"
    subtitle="subtitle xyz"
    description="description xyz"
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
