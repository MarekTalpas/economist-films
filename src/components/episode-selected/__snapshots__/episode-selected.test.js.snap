exports[`test EpisodeSelected renders with data 1`] = `
<div
  className="episode-selected">
  <img
    alt="title xyz"
    className="episode-selected__image"
    src="url xyz" />
  <div
    className="episode-selected__info-container">
    <div>
      EpisodeDescription
    </div>
    <div
      className="episode-buttons">
      <a
        className="episode-buttons__watch"
        onClick={[Function]}
        style={Object {}}>
        Watch Now
      </a>
      <a
        className="episode-buttons__learn"
        onClick={[Function]}
        style={Object {}}>
        Learn More
      </a>
    </div>
  </div>
</div>
`;
